extensions/audio/src/main/java/ai/djl/audio/dataset/AudioData.java

```diff
@@ -108,9 +107,9 @@ private float[] toFloat(String path) {
             logger.error(e.getMessage());
         }
         float[] floatArray = new float[list.size()];
-        int i = 0;
-        for (Float f : list) {
-            floatArray[i++] = (f != null ? f : Float.NaN);
+        for (int i = 0; i < list.size(); i++) {
```

Comment:
I think old one is better:
1. doesn't need to call list.size() every thime
2. it works better on non-randomaccess lists (Like LinkedList)

Reply:
Done